root/static/manifest.json

```diff
@@ -0,0 +1,99 @@
+{
+   "orientation" : "any",
+   "scope" : "/",
+   "start_url" : "/",
+   "theme_color" : "#da2037",
+   "icons" : [
+      {
+         "src" : "/static/manifest-icons/96x96-icon.png",
```

Comment:
I'm using small_logo.png but perhaps we want to use another. (just the dots but couldn't find it in the /images dir)